packer/build_image.sh

```diff
@@ -26,6 +27,7 @@ print_usage() {
     echo "  [--branch]              Set the release branch for GCE label. Default: master"
     echo "  [--ami-regions]         Set regions to copy the AMI when done building it (including permissions and tags)"
     echo "  [--build-tag]           Jenkins Build tag"
+    echo "  [--env-tag]             Environment tag for our images. default: daily for master and candidate for releases. Valid options: debug and production"
```

Comment:
Can you update the param description as well for default: debug

Reply:
done